internal/cluster/distribution/eks/eksadapter/node_pool_manager.go

```diff
@@ -16,27 +16,48 @@ package eksadapter
 
 import (
 	"context"
+	"strconv"
+	"strings"
 	"time"
 
 	"emperror.dev/errors"
 	"go.uber.org/cadence/client"
 
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+
 	"github.com/banzaicloud/pipeline/internal/cluster"
 	"github.com/banzaicloud/pipeline/internal/cluster/distribution/eks"
+	"github.com/banzaicloud/pipeline/internal/cluster/distribution/eks/eksprovider/workflow"
 	"github.com/banzaicloud/pipeline/internal/cluster/distribution/eks/eksworkflow"
+	"github.com/banzaicloud/pipeline/internal/global"
+	"github.com/banzaicloud/pipeline/internal/kubernetes"
+	"github.com/banzaicloud/pipeline/pkg/kubernetes/custom/npls"
 )
 
+// CF stack name prefix
+const StackNamePrefix = "pipeline-eks-nodepool-"
```

Comment:
Does this have to be exported? Everything should be unexported by default, unless needed somewhere else.